PyRDF/__init__.py

```diff
@@ -2,4 +2,22 @@
 from .Node import Node
 from .Proxy import Proxy
 from .Operation import Operation
-from .CallableGenerator import CallableGenerator
\ No newline at end of file
+from .CallableGenerator import CallableGenerator
+from .backend import Local, Dist
+
+Proxy.backend = Local({})
```

Comment:
Two comments about this line:
- I would add a default value for `config` in the constructor of `Local` so that we do not need to specify an empty dict here.
- I would perhaps move that constructor (exactly the one in `Local`) to `Backend`, because we will always set the configuration in any backend. Then if we need to do something specific for `Local` or any other backend, we can always write another constructor in those classes, and call the constructor of the superclass from that constructor.

Reply:
@etejedor , sounds good. I'll make these changes 👍